react/AsidedLayout/AsidedLayout.less

```diff
@@ -0,0 +1,21 @@
+@import (reference) "~seek-style-guide/theme";
+
+@media only screen and (min-width:740px) {
```

Comment:
`740px` is not magic, it's the `@responsive-type-breakpoint` as specified by the design team. (should probably be renamed as it's more than type)